### PR TITLE
Issue 12

### LINKED ISSUE #12: Fix the Xor method
There is currently a method called `xor` that lives in `/lib/xor.js`.
It is incomplete and needs to be fixed!

### Acceptance Criteria:
1. Running `yarn test xor` results in tests passing.

#### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request!

### PR DESCRIPTION
**Pull request by hugoduraes**
Make sure these boxes are checked before submitting this pull request~!~ Thank you!!

- [X] The test of the method you have fixed is passing
- [X] You have written a new skeleton method for someone else to work on!

Fixes #12

### CODE REVIEW

**Comment by hugoduraes** [comment]
I must be blind but I can't find what you mentioned! :P

File: test/remainder.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+test('remainder of 2 / 2 to equal 0', () => {
+  expect(remainder(2, 2)).toBe(0);
+});
+
+test('remainder of 3 / 2 to equal 1', () => {
+  expect(remainder(3, 2)).toBe(1);
+});
+
+test('remainder of 2 / 3 to equal 1', () => {
```
Comment (BlakeGuilloud) [inline test/remainder.test.js]: to equal 2

File: test/remainder.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  expect(remainder(2, 3)).toBe(2);
+});
+
+test('remainder of 2 / 1 to equal 1', () => {
```
Comment (BlakeGuilloud) [inline test/remainder.test.js]: to equal 0

File: test/remainder.test.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  expect(remainder(2, 1)).toBe(0);
+});
+
+test('remainder of 2 / 0 to equal 1', () => {
```
Comment (BlakeGuilloud) [inline test/remainder.test.js]: to be NaN

**Comment by BlakeGuilloud** [comment]
Haha no worries! I went ahead and tagged the specific lines. Also I'm going to try to publish this to npm as often as I can, so if you ever want to see your work in action, the registry should be current with the codebase!

**Comment by hugoduraes** [comment]
I really was blind! Thanks

**Pull request merged**